include/seqan3/alignment/scoring/detail/simd_matrix_scoring_scheme.hpp

```diff
@@ -80,6 +84,7 @@ class simd_matrix_scoring_scheme
     static constexpr scalar_type score_for_padding_symbol = (is_global) ? 1 : -1;
 
 
+
     //!\brief The scoring scheme stored as a linear array.
```

Comment:
two new lines were not enough? :smile: 
```suggestion

    //!\brief The scoring scheme stored as a linear array.
```

Reply:
mind the gap 😏